### PR TITLE
Feature/add ability adding a new chats

### PR DESCRIPTION
**Pull request by 0leksandrBondar**
Added "MoreDialogWindow"
image

image

and made "NewChatWindow"

image

added ability create new chat trough  NewChatWindow

### CODE REVIEW

**Comment by 0leksandrBondar** [comment]
@Serhii-Nesteruk please provide review

File: Client/Source/AppGui/ClientPage/CorrespondenceView/InputMessageField.qml [hunk 1]
```diff
@@ -67,7 +66,7 @@ Rectangle
         icon.source: Theme.iconFolderPath
         background: Rectangle
         {
-            color: Theme.inputMessageFieldColor
+            color: fileButton.hovered ? "#434543" : Theme.inputMessageFieldColor
```
Comment (Serhii-Nesteruk) [inline Client/Source/AppGui/ClientPage/CorrespondenceView/InputMessageField.qml:69]: Add "#434543" to SharedProperties.qml

File: Client/Source/AppGui/ClientPage/SettingsBar/SettingsBar.qml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Button
+    {
+        id: moreButton
+        width: 50
+        height: 50
+        icon.source: Theme.iconMorePath
+        anchors.top: parent.top
+        background: Rectangle
+        {
+            color: moreButton.hovered ? "#434543" : Theme.correspondenceSceneBackgroundColor
```
Comment (Serhii-Nesteruk) [inline Client/Source/AppGui/ClientPage/SettingsBar/SettingsBar.qml:22]: Add #434543" to SharedProperties.qml

File: Client/Source/AppGui/MoreOptionPages/ChatOptionPages/AddNewChatPage.qml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        text: qsTr("Create new chat")
+        anchors.top: parent.top
+        anchors.topMargin : parent.height / 10
+        anchors.horizontalCenter: parent.horizontalCenter
+        color: Theme.labelColor
+        font.pointSize: 24
+        font.bold: true
+    }
+
+    // Button
```
Comment (Serhii-Nesteruk) [inline Client/Source/AppGui/MoreOptionPages/ChatOptionPages/AddNewChatPage.qml:28]: Delete comments

**Comment by Serhii-Nesteruk** [comment]
CR Done

**Pull request merged**